benchmark/distributed/parse-address.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
```

Comment:
Btw, maybe extract this part of the [cli/connect](https://github.com/metarhia/jstp/blob/master/lib/cli/line-processor.js#L73) function and use it both here and in cli, it also has a few good errors?